ceph/ceph_admin.py

```diff
@@ -224,6 +224,73 @@ def manage_hosts(self, **kwargs):
             # todo: add code for removing host
             raise NotImplementedError
 
+    def label_host(self):
+        node_mon = self.cluster.get_nodes(role="mon")
+        node_mgr = self.cluster.get_nodes(role="mgr")
+        node_osd = self.cluster.get_nodes(role="osd")
+        node_mds = self.cluster.get_nodes(role="mds")
+        node_rgw = self.cluster.get_nodes(role="rgw")
+        node_client = self.cluster.get_nodes(role="client")
+        # self.ceph_iscsi(self.cluster.get_nodes(role="iscsi"))
```

Comment:
Could you remove the code that is commented? We can have comments but it would be good to avoid code that is commented.